virt/release_notes/virt-4-20-release-notes.adoc

```diff
@@ -73,10 +72,12 @@ This release adds new features and enhancements related to the following compone
 // CNV-62399
 * The descheduler profile `DevKubeVirtRelieveAndMigrate` has been renamed to `KubeVirtRelieveAndMigrate` and is now generally available. The updated profile improves VM eviction stability during live migrations by enabling background evictions and reducing oscillatory behavior. For more information, see xref:../../virt/managing_vms/advanced_vm_management/virt-enabling-descheduler-evictions.adoc#virt-configuring-descheduler-evictions_virt-enabling-descheduler-evictions[Configuring descheduler evictions for virtual machines].
 
+// CNV-30390
+* vNUMA topology for VMs is now generally available (GA) and is tested and supported for production use. Before this update, this feature was provided as a Technology Preview, and KubeVirt VMs were not optimally configured for NUMA. This could lead to potential performance degradation due to ignorance of the host system's NUMA topology. With this update, the NUMA configuration for VMs is improved, ensuring better performance and optimal resource allocation. For more information, see xref:../../virt/managing_vms/advanced_vm_management/virt-numa-topology.adoc#virt-numa-topology[Working with NUMA topology for virtual machines].
```

Comment:
```suggestion
* vNUMA topology for VMs is now generally available (GA) and is tested and supported for production use allowing users to opt-in to a better NUMA configuration for VMs, with better performance and optimal resource allocation. For more information, see xref:../../virt/managing_vms/advanced_vm_management/virt-numa-topology.adoc#virt-numa-topology[Working with NUMA topology for virtual machines].
```
`vNUMA` configuration remains opt-in per VM so the fact this feature is now GA doesn't automatically enable it for all VMs and resolve all perf issues etc.